students/KyleBarry/lesson03/basic_operations.py

```diff
@@ -17,7 +45,7 @@ def add_customer(customer_id, first_name, last_name, home_address, phone_number,
                                status=status,
                                credit_limit=credit_limit)
         cust.save()
-        logger.info("Customersuccessfully added!")
+        logging.info("Customer successfully added!")
 
     except Exception as e:
         logging.info(f"Could not add {first_name} to db")
```

Comment:
It is best to catch the specific exception that you are expecting, like you did above. Removing the try/except block and causing the exception to happen on purpose is a good way to figure out what exception you are looking for. Also you should log the exception itself. ```logging.error(e)```